content/working_with/console/pages/local-blueprints-page.md

```diff
@@ -1,22 +1,33 @@
 ---
 layout: bt_wiki
-title: Local Blueprints Page
+title: Blueprints Page
 category: Console
 draft: false
 abstract: Blueprint Page Reference
 weight: 130
 aliases: ["/manager_webui/blueprints-page/", "/working_with/console/blueprints-page/", "/working_with/console/local-blueprints-page/"]
 ---
 
-The Local Blueprints page acts as the catalog of services (blueprints) that are available for use on this {{< param cfy_console_name >}}. It provides a list of the blueprints uploaded to the {{< param cfy_manager_name >}} and action buttons to upload, deploy and delete them. It also allows to open [{{< param cfy_composer_name >}}]({{< param cfy_composer_link >}}) or edit specific blueprint in [{{< param cfy_composer_name >}}]({{< param cfy_composer_link >}}).
+The Blueprints page acts as the catalog of services (blueprints) that are available for use on this {{< param cfy_console_name >}}.
+It provides a list of the blueprints uploaded to the {{< param cfy_manager_name >}} and action buttons to upload, deploy and delete them.
+It also allows to edit specific blueprint in [{{< param cfy_composer_name >}}]({{< param cfy_composer_link >}}).
 
 {{% note %}}
-[{{< param cfy_composer_name >}}]({{< param cfy_composer_link >}}) is only available in [{{< param mgr_premium_title >}}]({{< param mgr_premium_link >}}).
+The "edit in [{{< param cfy_composer_name >}}]({{< param cfy_composer_link >}})" option is only available in [{{< param mgr_premium_title >}}]({{< param mgr_premium_link >}}).
 {{% /note %}}
 
-![Local Blueprints Page]( /images/ui/pages/local-blueprints-page.png )
+![Blueprints Page]( /images/ui/pages/local-blueprints-page.png )
 
-For more information check documentation pages for widgets used in this page:
+## Blueprint marketplace
 
-* [Blueprints]({{< relref "working_with/console/widgets/blueprints.md" >}})
-* [{{< param cfy_composer_name >}} link]({{< relref "working_with/console/widgets/composerLink.md" >}})
+In addition to uploading a blueprint from a specified package it is possible to upload one of the blueprints predefined in the **Marketplace**.
+
+The **Marketplace** contains blueprints organized in categories:
+* AWS
```

Comment:
It renders the following way:
![image](https://user-images.githubusercontent.com/5202105/149103114-5a957292-d103-41c9-823c-c1064e9b5b72.png)

I suspect you just need to add empty line to fix it:

```suggestion
The **Marketplace** contains blueprints organized in categories:

* AWS
```